domain/src/main/java/com/anytypeio/anytype/domain/page/CreateObject.kt

```diff
@@ -50,7 +46,7 @@ class CreateObject(
         }
 
         val command = Command.CreateObject(
-            template = template,
+            template = null,
```

Comment:
so template is never applied when creating an object?

Reply:
In the context of this UseCase, if the type has at least one template, we show the templates menu; otherwise, an object without a template is created.